### PR TITLE
pyproject: add all dependencies required for full build with tests

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Comment by smarie** [comment]
pypandoc/pandoc is actually not useful, I'll update travis accordingly. Can you please update your PR too?

File: pyproject.toml [hunk 1]
```diff
@@ -3,5 +3,12 @@ requires = [
     "setuptools",
     "setuptools_scm",
     "six",
+    "pytest-runner",
+    "pytest-harvest",
+    "pypandoc",
```
Comment (smarie) [inline pyproject.toml]:
```suggestion
```

**Comment by keszybz** [comment]
Updated.

**Comment by smarie** [comment]
I fixed an issue happening on the python 3.5 targets because of pip 10 (https://github.com/gevent/gevent/issues/1180 )

**Pull request merged**